docs/tutorials/contracts/nfts/blockcraft-nfts.md

```diff
@@ -0,0 +1,243 @@
+
+## Prerequisites
+
+> **Tip:** This tutorial is meant to show a cool example that builds off of our previous [NFT tutorial](/docs/tutorials/contracts/nfts/minting-nfts). If you haven't gone through it, we highly recomming doing so.
+
+To complete this tutorial successfully, you'll need:
+
+- A Minecraft account
+- WorldEdit [installed](https://worldedit.enginehub.org/en/latest/install/)
+- [A NEAR account](#wallet)
+- [NEAR command-line interface](/docs/develop/contracts/rust/intro#installing-the-near-cli) (`near-cli`)
+
+## Creating a Schematic
+
+In this section, we'll be creating the schematic to put on chain.
+
+### Setup
+
+Once you have WorldEdit installed and Minecraft loaded up, let's check if WorldEdit is working properly by running:
+
+```bash
+//pos1
+```
+
+If WorldEdit is properly installed, it should output `First position set to (X, Y, Z).` where X, Y, and Z are coordinates.
+
+For this tutorial, we will be minting a small village house. To follow along, choose any structure that you'd like to mint as shown below:
+
+> ![Village House Minecraft](/docs/assets/nfts/village-house-minecraft.png)
+
+You'll then want to choose the boundaries of the structure that you'd like to copy. We will turn these into schematics which will be placed on chain for you or others to potentially download and paste in your own worlds.
+
+To do this, we'll need to outline the boundaries of the build using WorldEdit. Stand in the bottom left corner of your build and run:
+
+```bash
+//pos1
+```
+
+You can then move to the top right corner and run:
+
+```bash
+//pos2
+```
+
+Behind the scenes, this has created a cube around your build. We can then copy what we just selected using `//copy`. The output should look something like this:
+
+> ![Copy Chat Message](/docs/assets/nfts/copy-chat-message-minecraft.png)
+
+> **Tip:** Your player position matters when copying. If you copy the build and you're standing on the roof, when you paste the build, it will paste in a way that will result in you standing on the roof.
+
+### Sanity Check
+
+We can check and see if our build is fine by pasting what we just copied elsewhere in our world. Go to a location that you would like to paste the build and run `//paste`. In our case, we pasted the village house floating above a coral reef biome:
+
+> ![Pasted Minecraft House](/docs/assets/nfts/pasted-minecraft-house.png)
+
+### Creating the Schematics File
+
+Now that you're happy with the build you've just copied and pasted, it's time to create the schematic file that we'll put on chain. To do this we'll run a WorldEdit command that will save the schematic file on our local machine. Run the command `schematic save FILE_NAME`. This will save the file to your minecraft folder with the file name you passed in.
```

Comment:
```suggestion
Now that you're happy with the build you've just copied and pasted, it's time to create the schematic file that we'll put on-chain. To do this we'll run a WorldEdit command that will save the schematic file on our local machine. Run the command `schematic save FILE_NAME`. This will save the file to your Minecraft folder with the file name you passed in.
```